P/PThreadPool/build_tarballs.jl

```diff
@@ -36,14 +43,16 @@ install_license ../LICENSE
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
 platforms = supported_platforms()
+filter!(p -> !(Sys.isapple(p) && arch(p) == "aarch64"), platforms) # aarch64-macos unsupported for CPUInfo 20200522
```

Comment:
It actually *does* build on `aarch64-apple`, but the proper `CPUInfo` version (https://github.com/Maratyszcza/pthreadpool/blob/029c88620802e1361ccf41d1970bd5b07fd6b7bb/cmake/DownloadCpuinfo.cmake#L7) that should be used for building does not support aarch64-apple - but it also seems CPUInfo is not even being used...